build_tools/buildkite/cmake/android/arm64-v8a/pipeline.yml

```diff
@@ -18,22 +18,24 @@ steps:
 
   - wait
 
-  - label: "test on Galaxy S20 (exynos-990, mali-g77)"
+  - label: "test on Pixel 6 (google-tensor, mali-g78)"
     commands:
+      - "git clean -fdx"
```

Comment:
what's this for? seems unrelated to changing which phone is used?

Reply:
It's cleaning up previously downloaded artifacts on RPIs to avoid seeing out of storage issue. Yes not related to phone change. But trivial enough to bundle (as I don't think we will have issues with this change so need separate reverting). I added a note in the commit message.